app/src/main/java/org/wikipedia/analytics/eventplatform/CustomizeToolbarEvent.kt

```diff
@@ -0,0 +1,30 @@
+package org.wikipedia.analytics.eventplatform
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import org.wikipedia.auth.AccountUtil
+
+@Suppress("unused")
+@Serializable
+@SerialName("/analytics/mobile_apps/android_customize_toolbar_interaction/1.0.0")
+class CustomizeToolbarEvent(private var is_anon: Boolean,
+                            private var is_rfm_enabled: Boolean,
+                            private var source: String,
+                            private var favorites_order: List<Int>,
+                            private var menu_order: List<Int>,
+                            private var time_spent_ms: Int) : TimedEvent(STREAM_NAME) {
+
+    fun logCustomization(source: String, favorites_order: List<Int>, menu_order: List<Int>, is_rfm_enabled: Boolean) {
```

Comment:
Would it be possible to make the parameters have a naming style like this: `favoritesOrder`?

Reply:
The parameter names have to match the MEP tables Cooltey ... and they have a snake_case convention..